manifests/assisted-image-service/assisted-image-service-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: assisted-installer
+  name: assisted-image-service
+  annotations:
+    cert-manager.io/issuer: assisted-installer-ca
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+spec:
+  rules:
+  - host: assisted-image-service-assisted-installer.apps.dev.redhat.com
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: assisted-image-service
+            port:
+              name: https
```

Comment:
I ran into this too and ended up just [using port numbers instead](https://github.com/openshift/assisted-service/pull/6446/files#diff-c619bb36181b5e1dd76fb1fcc1f54068b16a8761f77106052fcbf5e2f5f9d356R927-R929) :shrug: